projects/1.19/assets/quark/quark/lang/zh_cn.json

```
@@ -1,1296 +1,3379 @@
+
+	"quark.config.building.midori.name": "绿",
+	"quark.config.building.midori.desc": "绿是紫珀的绿色对应物，用苔藓泥制成，配方与紫珀系列相同。\n苔藓泥是一种新物品，通过在熔炉中熔炼苔藓块获得。\n（原文为Midori，在日语中意为「绿色」）\n注意：在1.18前的版本，苔藓泥被称作仙人掌泥，作为仙人掌的烧炼产物取代了绿色染料。它能够按1比1的比例制成绿色染料。",
+
+	"quark.config.building.more_brick_types.name": "更多砖",
+	"quark.config.building.more_brick_types.desc": "为一些方块添加了新的砖变种：\n(1) 砂岩砖（适用于所有类型的砂岩）可以用切制砂岩制作。\n(2) 圆石砖及其苔藓变种的制作方法与石砖相同，只是使用圆石。将圆石砖与其他方块（1.16+）组合，可以制造它们的砖变种。\n(3) 黑石砖可以用圆石砖和黑石制作。\n(4) 泥土砖可以用圆石砖和泥土制作。\n(5) 下界岩砖可以用圆石砖和下界岩制作。\n(6) 蓝色下界砖可以用下界砖和诡异疣块制作。红色下界砖也添加了类似的配方。\n(7) 此外，所有这些都有台阶、楼梯和墙的变种。",
+	"quark.config.building.more_brick_types.enable_blackstone_bricks.name": "启用黑石砖",
+	"quark.config.building.more_brick_types.enable_blackstone_bricks.desc": "需要先启用圆石砖",
+	"quark.config.building.more_brick_types.enable_blue_nether_bricks.name": "启用蓝色下界砖",
+	"quark.config.building.more_brick_types.enable_blue_nether_bricks.desc": "此项还附带了红色下界砖的实用配方",
+	"quark.config.building.more_brick_types.enable_cobblestone_bricks.name": "启用圆石砖",
+	"quark.config.building.more_brick_types.enable_cobblestone_bricks.desc": "此项还包括苔圆石砖",
+	"quark.config.building.more_brick_types.enable_dirt_bricks.name": "启用泥土砖",
+	"quark.config.building.more_brick_types.enable_dirt_bricks.desc": "需要先启用圆石砖",
+	"quark.config.building.more_brick_types.enable_netherrack_bricks.name": "启用下界岩砖",
+	"quark.config.building.more_brick_types.enable_netherrack_bricks.desc": "需要先启用圆石砖",
+	"quark.config.building.more_brick_types.enable_sandstone_bricks.name": "启用砂岩砖",
+	"quark.config.building.more_brick_types.enable_sandstone_bricks.desc": "此项还包括红砂岩砖和灵魂砂岩砖",
+
+
+	"quark.config.building.more_mud_blocks.name": "更多泥巴方块",
+	"quark.config.building.more_mud_blocks.desc": "有一些新的泥巴建筑方块，包括网格状的泥砖、泥柱和雕刻过的泥砖。",
+
+	"quark.config.building.more_potted_plants.name": "更多盆栽",
+	"quark.config.building.more_potted_plants.desc": "现在又有17种原版方块可以做成盆栽了。\n17种方块包括：丁香、牡丹、玫瑰丛、向日葵、草、可可豆、甘蔗、甜浆果、紫颂花、海泡菜、小麦、马铃薯、胡萝卜、甜菜根、南瓜、西瓜、下界疣。\n（小麦、甜菜根、南瓜、西瓜的盆栽是使用对应的种子制作的。）\n1.16.3+：额外添加了高草丛、大型蕨、藤蔓、缠怨藤、垂泪藤、下界苗的盆栽。",
+
+	"quark.config.building.more_stone_variants.name": "更多石头变种",
+	"quark.config.building.more_stone_variants.desc": "原版和夸克添加的所有类型的石头都有砖、錾制砖和柱的变种。\n砖有台阶、楼梯和墙的变种，每种石头的錾制砖变种看起来都不同。",
+	"quark.config.building.more_stone_variants.enable_bricks.name": "启用砖变种",
+	"quark.config.building.more_stone_variants.enable_chiseled_bricks.name": "启用錾制砖变种",
+	"quark.config.building.more_stone_variants.enable_pillar.name": "启用柱变种",
+
+	"quark.config.building.nether_brick_fence_gate.name": "下界砖栅栏门",
+	"quark.config.building.nether_brick_fence_gate.desc": "用下界砖块制成的栅栏门。用下界砖块代替木板，用下界砖代替木棍制作。",
+
+	"quark.config.building.rainbow_lamps.name": "彩虹灯",
+	"quark.config.building.rainbow_lamps.light_level.name": "光照等级",
+	"quark.config.building.rainbow_lamps.use_corundum.name": "使用刚玉",
+	"quark.config.building.rainbow_lamps.use_corundum.desc": "已启用刚玉模块时，彩虹灯是否应该由刚玉制成，且使用刚玉主题。",
```

```suggestion
	"quark.config.building.rainbow_lamps.use_corundum.desc": "控制在已启用刚玉模块时，彩虹灯是否应该由刚玉制成，且使用刚玉主题的外观。",
```
“刚玉主题”，语义没错，但为什么读起来这么别扭

---

加个 `控制在` 很别扭，本身就是一个开关，玩家只需要知道它控制着什么就行了

---

合理，改了